CHANGES.md

```diff
@@ -138,6 +139,34 @@ OpenSSL 3.1
 
    *Tomáš Mráz*
 
+ * Various fixes and extensions to certificate handling and the `x509` etc. apps
+
+   *David von Oheimb*
+
+ * Various fixes and extensions to the CMP+CRMF implementation and the `cmp` app
+
+   *David von Oheimb*
+
+ * Fixes and extensions to the HTTP client and to the HTTP server in `apps/`
+
```

Comment:
I am not sure these entries are really useful. If these are really just fixes or minor API additions, IMO they do not need a CHANGES entry.

Reply:
I've made these entries more specific by adding some detail.
There are several extensions towards CMPv3, and
I believe that it will be good that users also know for instance about the upcoming tracing capabilities of cert chain building and HTTP client+server.